src/main/java/core/screens/GameScreen.java

```diff
@@ -68,6 +84,16 @@ private void handleUserInput() {
         if (Gdx.input.isKeyPressed(Input.Keys.UP)) {
             y += inc;
         }
+        if (Gdx.input.isKeyJustPressed(Input.Keys.I)){
+            displayInventory = !displayInventory;
+            if (displayInventory){
+                hud.openInventory(inventory);
+            }else{
+                hud.closeInventory(inventory);
+            }
+        }
```

Comment:
If leaving i/o code in GameScreen/GameplayController, add TODO comment to remind us to move this to the PlayerManager or wherever appropriate

Reply:
Right, added TODO